src/test/unit/inboundApi/handlers.test.js

```diff
@@ -36,14 +44,21 @@ describe('Inbound API handlers:', () => {
                 },
                 response: {},
                 state: {
-                    conf: {}
+                    conf: {},
+                    logger: new Logger({ context: { app: 'inbound-handlers-unit-test' }, space: 4, transports: logTransports })
                 }
             };
         });
+
         test('calls `model.quoteRequest` with the expected arguments.', async () => {
             const quoteRequestSpy = jest.spyOn(Model.prototype, 'quoteRequest');
 
-            await handlers.map['/quotes'].post(mockContext);
+            try {
+                await handlers.map['/quotes'].post(mockContext);
+            }
+            catch(e) {
+                expect(e).toBe(undefined);
```

Comment:
So there is no error object here?

Reply:
this is an accepted pattern actually in jest docs but I have changed it to the alternative one-liner.